src/styles/_variables.scss

```diff
@@ -1,10 +1,37 @@
 /*
+ * Base Settings
+ */
+
+$margin-vertical-sm: 1.666666667rem !default;
+$margin-vertical-lg: 5.555555556rem !default;
+
+$base-line-height:   1.555rem !default; // 28px
+$base-font-size:     18px !default;
+
+$gutter-width: 15px !default;
+
+$font-family-body:    "proxima-nova", sans-serif !default;
+$font-family-heading: "proxima-nova", sans-serif !default;
+$font-family-code:    "Consolas", "Monaco", "Andale Mono", "Ubuntu Mono", monospace !default;
+
+// Image Path.
+$images-path: "../images" !default;
```

Comment:
This should be set to `$hm-images` so it can be safely overridden in other projects